beginend.el

```diff
@@ -239,6 +239,13 @@ BEGIN-BODY and END-BODY are two `progn' expressions passed to respectively
   (progn
     (prodigy-last)))
 
+(beginend-define-mode magit-status-mode
+  (progn
+    (re-search-forward "^$")
+    (forward-line))
+  (progn
+    (beginend--goto-nonwhitespace)))
```

Comment:
Can you tell me how to get a status buffer where `(beginend--goto-nonwhitespace)` doesn't work?